Makefile

```diff
@@ -110,7 +110,12 @@ openshift-client:
 	@type -p oc > /dev/null || bash hack/get-openshift-client.sh
 
 .PHONY: build
-build: bin/cluster-logging-operator
+build: bin/cluster-logging-operator build-functional-e2e-tests
+
+.PHONY: build-functional-e2e-tests
+build-functional-e2e-tests:
+	for d in $$(find ./test/e2e -type d -name "*"); do go test -c $$d --tags vector fluentd; done
+	for d in $$(find ./test/functional -type d -name "*"); do go test -c $$d --tags vector fluentd; done
```

Comment:
Did you mean to set both vector and fluentd tags? The syntax is comma-separated `--tags vector,fluentd`, but that doesn't work as they are not meant to be set together:
```
test/functional/vector.go:9:7: LogCollectionType redeclared in this block
	test/functional/fluentd.go:9:7: other declaration of LogCollectionType
```